js/src/View/AppView.js

```diff
@@ -0,0 +1,87 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var UserModel = require('../Model/UserModel.js');
+var appViewTemplate = require('../templates/container.hbs');
+var TodoListView = require('./TodoListView.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoListModel = require('../Model/TodoListModel.js');
+var todoAppTemplate = require('../templates/todoapp.hbs');
+
+var AppView = Backbone.View.extend({
+    el: 'body',
+    events: {
+        'click .addlist': 'addNewList',
+    },
+
+    initialize: function(){
+
+        this.render();
+
+        this.user = new UserModel();
+        this.user.fetch({
+            success: _.bind(function(model){
+                console.log('Success: user loaded.');
+                this.renderUser();
+            }, this),
+            error: function(){
+                console.log('Error: user failed to load.');
+            }
+        })
+
+        this.todoLists = new TodoListCollection();
+        this.todoLists.fetch({
+            success: _.bind(function(model){
+                console.log('Success: todoLists loaded.');
+                this.renderTodoLists();
+            }, this),
+            error: function(){
+                console.log('Error: todolists failed to load.');
+            }
+        });
+    },
+
+    render: function(){
+        this.el.insertAdjacentHTML('afterbegin', todoAppTemplate({}));
+    },
+
+    renderUser: function(){
+        this.el.querySelector('.user-container').innerHTML = appViewTemplate({userName: this.user.get('name')});
```

Comment:
Update references to `user-container` to `js-user-container`. If you want to quickly find references to it, you can use `git grep <sort of regex pattern>` in your terminal to find the specified pattern. `git grep user-container`.